bpf/README.md

```diff
@@ -0,0 +1,69 @@
+# BPF 101
+
+If you're reading this, you were probably tasked with writing a BPF code.
+This document will help you get started.
+
+#### Before you start here are some useful links:
+
+https://docs.kernel.org/bpf/ - BPF documentation
+https://github.com/aquasecurity/libbpfgo - Go bindings for libbpf
+https://github.com/iovisor/bcc - BPF Compiler Collection (a lot of examples)
+https://github.com/cilium/cilium - BPF-based networking, security, and observability (more examples)
+https://man7.org/linux/man-pages/man7/bpf-helpers.7.html - BPF helpers documentation
```

Comment:
```suggestion
* https://docs.kernel.org/bpf/ - BPF documentation
* https://github.com/aquasecurity/libbpfgo - Go bindings for libbpf
* https://github.com/iovisor/bcc - BPF Compiler Collection (a lot of examples)
* https://github.com/cilium/cilium - BPF-based networking, security, and observability (more examples)
* https://man7.org/linux/man-pages/man7/bpf-helpers.7.html - BPF helpers documentation
```

Markdown joins the lines otherwise.